### ISSUE TITLE
Save accent of skipped sentences

### ISSUE THREAD

**Issue by danielsjf** [mozilla/voice-web#1151]
One of the reasons why people skip sentences might be because the sentence is rarely used in their accent. For instance some British expression that is not used in American English might be skipped by Americans, but not by British.

**Comment by Gregoor** [comment]
Good point! Fortunately we save the reference to the sentence and the user, from which we could infer the accent.

**Comment by ftyers** [comment]
This is basically now supported through the variant system. Languages can now define variants and people can choose to contribute only to that variant.
